test/ITensorLegacyMPS/base/runtests.jl

```diff
@@ -14,3 +14,5 @@ ITensors.disable_threaded_blocksparse()
     @time include(filename)
   end
 end
+
+nothing
```

Comment:
I like to put `nothing` at the end of testing files so that they don't print a very large output when I run them locally.

Reply:
Can't you just run it as `include("runtests.jl");`?

Reply:
I did run it that way, but on my machine it prints a huge output at the end. Most of the output seems related to broken tests but I'm not sure if that's the main reason. I think it's because the Test module returns some kind of data structure that gets displayed in the REPL afterward.

Reply:
The `;` at the end should suppress output to the REPL.

Reply:
Oh I see. Then yes no problem to run it that way.

Reply:
I removed those `nothing` statements from all the tests and `runtests.jl` in `ITensorMPS`.